### ISSUE TITLE
[ Softmax() missing ]

### ISSUE THREAD

**Issue by IemProg** [jeonsworld/ViT-pytorch#41]
Thanks for sharing the ViT implementation, wonderful work.

I'm wondering why you are not using the Softmax() function after head component -from features to classes-, as you did in the Jupyter notebook example?

Thanks
